kernel/src/arch/amd64/interrupts/pit.cpp

```diff
@@ -10,32 +10,25 @@
 
 #include "pit.hpp"
 
-using namespace pit;
+using namespace arch::amd64;
 
-pit::Pit timer = Pit();
+Pit arch::amd64::timer = Pit();
 
-void Pit::update() {
-	auto c = Pit::command_t {
-		.bcd = 0,
-		.mode = this->mode,
-		.access = access_t::LOBYTE_HIBYTE,
-		.channel = 0
-	};
-	command.write(*reinterpret_cast<uint8_t*>(&c));
+void Pit::update() noexcept {
+	Pit::command_t c{.bcd = 0, .mode = this->mode, .access = access_t::LOBYTE_HIBYTE, .channel = 0};
+	command.write(*reinterpret_cast<uint8_t *>(&c));
```

Comment:
Can this made `static_cast`?

Reply:
No - `command_t` is a struct packed into a `u8` so needs to be pointer aliased
Although maybe would be better to use `std::bitcast`